lib/rubocop/sorbet/inject.rb

```diff
@@ -11,6 +11,14 @@ class << self
         def defaults!
           path = CONFIG_DEFAULT.to_s
           hash = ConfigLoader.send(:load_yaml_configuration, path)
+          if RuboCop::Version::STRING >= "1.66"
```

Comment:
Is this the customary way to check the Rubocop version? I'm worried it might not work in some edge cases (because it'll do string based ordering, not numerical ordering)

I asked about this in the Rubocop Discord: https://discord.com/channels/797752368217063424/797752368217063427/1280883199794483280